src/sections/Home/components/Fullname/HomeFullname.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState, useEffect } from "react";
+import { styles } from "../../../../styles/styles";
+import { makeStyles, Typography, Fade } from "@material-ui/core";
+import AnimatedRole from "../AnimatedRole/HomeRole";
+
+const useStyles = makeStyles(theme => ({
+  homeCont: {
```

Comment:
same comment about styles here and everywhere, I believe you can use alot of the rules you have defined already